vnext/Desktop.UnitTests/MemoryMappedBufferTests.cpp

```diff
@@ -0,0 +1,149 @@
+#include "MemoryMappedBuffer.h"
+#include "utilities.h"
+
+#include <CppUnitTest.h>
+
+#include <shlwapi.h>
+#include <windows.h>
+
+#include <cstring>
+#include <memory>
+
+using facebook::jsi::Buffer;
+using facebook::jsi::JSINativeException;
+using facebook::react::utilities::checkedReinterpretCast;
+using Microsoft::JSI::makeMemoryMappedBuffer;
+using Microsoft::VisualStudio::CppUnitTestFramework::Assert;
+
+namespace {
+
+static constexpr const wchar_t *const testFileName = L"temp.txt";
+
+static void writeTestFile(const char *const content, size_t size) {
+  FILE *testFilePtr;
+
+  // We need pass in the "b" flag to ensure that fwrite modify its input when
+  // outputting to disk.
+  if (_wfopen_s(&testFilePtr, testFileName, L"wb")) {
+    return;
+  }
+
+  std::unique_ptr<FILE, decltype(&fclose)> testFilePtrWrapper(
+      testFilePtr, fclose);
+
+  fwrite(content, sizeof(*content), size, testFilePtrWrapper.get());
+}
+
+static void deleteTestFile() {
+  if (!DeleteFileW(testFileName)) {
+    Assert::IsTrue(GetLastError() == ERROR_FILE_NOT_FOUND);
+  }
+  Assert::IsTrue(!PathFileExistsW(testFileName));
+}
+
+uint32_t getPageSize() noexcept {
+  SYSTEM_INFO systemInfo;
+  GetSystemInfo(&systemInfo);
+  return systemInfo.dwPageSize;
+}
+
+} // namespace
+
+// Clang format does not work with the Microsoft Cpp Test Macros. Hence we turn
+// it off here.
+// clang-format off
+
+TEST_CLASS(MemoryMappedBufferUnitTests){
+ private:
+   TEST_METHOD_INITIALIZE(initialize) {
+     deleteTestFile();
+   }
+
+   TEST_METHOD_CLEANUP(cleanUp) {
+     deleteTestFile();
+   }
+
+ public :
+  TEST_METHOD(SimpleTest_NoOffest) {
```

Comment:
>SimpleTest_NoOffest [](start = 14, length = 19)

Please be consistent with naming, some are Pascal (SimpleTest_NoOffset), some are Camel (cleanUp)

Reply:
I went ahead ended up using PascalCase for everything.

---
In reply to: [320917305](https://github.com/microsoft/react-native-windows/pull/3059#discussion_r320917305) [](ancestors = 320917305)